lib/client-assets.php

```diff
@@ -1573,14 +1573,27 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		'maxUploadFileSize'      => $max_upload_size,
```

Comment:
It works!

![image](https://user-images.githubusercontent.com/36432/47562940-12cf2880-d8d5-11e8-8449-2d6bd7d730f4.png)

Should I be seeing that error message though?

Reply:
I saw the error mentioned for other meta boxes elsewhere. I wonder if it's a more generic bug in 5.0?

Reply:
How did you get this error?